### PR TITLE
Add properties to runserver.Command

### PR DESCRIPTION
**Pull request by Photonios**
See: https://github.com/django/django/blob/master/django/core/management/commands/runserver.py#L27

### CODE REVIEW

File: django-stubs/core/management/commands/runserver.pyi [hunk 1]
```diff
@@ -1,3 +1,12 @@
+from typing import Tuple
+
 from django.core.management.base import BaseCommand
 
-class Command(BaseCommand): ...
+class Command(BaseCommand):
+    requires_system_checks: bool
+    stealth_options: Tuple[str]
```
Comment (sobolevn) [inline django-stubs/core/management/commands/runserver.pyi]: Should be `Tuple[str, ...]` to have the same type as other commands. For example: https://github.com/django/django/blob/eac9ab7ebb1ce0cbbc79c4cf65e8f70b0635a240/django/core/management/commands/flush.py#L15

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks a lot!

File: django-stubs/core/management/commands/runserver.pyi (hunk 1 above)
Comment (Photonios) [reply django-stubs/core/management/commands/runserver.pyi]: Should this definition go into `BaseCommand` then?

Comment (sobolevn) [reply django-stubs/core/management/commands/runserver.pyi]: Yes, I think so: https://github.com/django/django/blob/eac9ab7ebb1ce0cbbc79c4cf65e8f70b0635a240/django/core/management/base.py#L212-L226

Please, check other props as well.

Comment (Photonios) [reply django-stubs/core/management/commands/runserver.pyi]: I've updated the type for `stealth_options` and `base_stealth_options` in `BaseCommand` as they were declared as `Any`.

File: django-stubs/core/management/commands/runserver.pyi [hunk 2]
```diff
@@ -1,3 +1,7 @@
 from django.core.management.base import BaseCommand
 
-class Command(BaseCommand): ...
+class Command(BaseCommand):
+    default_addr: str
```
Comment (Photonios) [inline django-stubs/core/management/commands/runserver.pyi]: Looking at other definitions, I believe this should be `default_addr: str = ...`.. Correct?

(Same applies to the other ones I've added here)

Comment (sobolevn) [reply django-stubs/core/management/commands/runserver.pyi]: Yes, this is a style thing mostly. Sorry, we don't have a way to check that automatically for now.

Comment (Photonios) [reply django-stubs/core/management/commands/runserver.pyi]: No problem. Done :+1:

**Review by sobolevn — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**